selinux/services/udev.cil

```diff
@@ -0,0 +1,33 @@
+(type udev_exec_t)
+(call system_f (udev_exec_t))
+(context udev_exec_t (system_u object_r udev_exec_t (systemLow systemLow)))
+(filecon "/sbin/systemd-udevd" file udev_exec_t)
+(filecon "/usr/bin/udevadm" file udev_exec_t)
+(filecon "/sbin/udevadm" symlink udev_exec_t)
+
+; Do not reorder: label non-executable rules files as executable
+(type udev_rules_t)
+(call system_f (udev_rules_t))
+(filecon "/usr/lib/udev/rules.d(/.*)?" any (system_u object_r udev_rules_t (systemLow systemLow)))
+; TODO: separate label to be called by udev only?
+(filecon "/usr/lib/udev(/.*)?" any udev_exec_t)
+
+(type udev_t)
+(call system_service_p (udev_t))
+(allow init_t udev_exec_t (file (execute)))
+(allow udev_t udev_exec_t (file (execute)))
+
+; execute is standard for systemd-udevd, execute_no_trans for udevadm called in init scope
+(allow init_t udev_exec_t (file (execute_no_trans)))
+
+; cap-launcher
+(allow udev_t udev_exec_t (file (entrypoint execute_no_trans)))
+
+; TODO: label /run
+
+; modprobe called by kernel or udev
```

Comment:
`vgchange` can also modprobe

Reply:
Unsure, maybe I will do policies for it later. Or we just let it `execute_no_trans` and run in init's domain.

Reply:
Implemented it in the way misc sbin things can be ran by machined, and run in init context. We should also take care of the cases when udev runs some binaries not stored in its rules directory

Reply:
yeh, udev rules can execute binaries on host and if we block that it might break tools

Reply:
We need to figure out which binaries